### ISSUE TITLE
Not working 4.7-Navigate_To_menu_related_files.html

### ISSUE THREAD

**Issue by sun-flat-yamada** [JetBrains/resharper-rider-samples#36]
# Summary

- Not working `01-Navigation/4-Contextual_navigation/4.7-Navigate_To_menu_related_files.html`
- When I followed the instructions and executed the "Navigate to (Ctrl+Shift+G)" shortcut in the HTML comment, nothing responded.
- I want to see what your expectations are.

# Glance at the problem code

```html
﻿<!DOCTYPE html>
<html>
<head>
    <title>Navigate To menu - related files</title>


    <!-- Navigate To menu
         
         Displays a contextual menu of options you can use to navigate to from
         your current location
         
         Very useful way of navigating without having to learn ALL of the shortcuts!
         
         <shortcut id="Navigate To...">Alt+`</shortcut>
                  
         
         1. Navigate to related files. E.g. referenced files from HTML
            Put text caret here →
            Navigate to Related Files shows list of other files containing definitions of PartialClass
        -->
    <script type="text/javascript" src="ExampleCode/script1.js"></script>
    <script type="text/javascript" src="ExampleCode/script2.js"></script>
    <script type="text/javascript" src="ExampleCode/script3.js"></script>

    <link rel="stylesheet" type="text/css" href="ExampleCode/style1.css">
    <link rel="stylesheet" type="text/css" href="ExampleCode/style2.css">
    <link rel="stylesheet" type="text/css" href="ExampleCode/style3.css">
</head>
<body>

</body>
</html>
```

# Environment

- Rider 2021.1.2
- Keymap : Default IntelliJ Style

**Comment by maartenba** [comment]
That's a ReSharper-only feature in this file. You could try this one in Rider:

```
         <shortcut id="Navigate To Related Symbol">Ctrl+Alt+F7</shortcut> (Rider Visual Studio keymap)
```

Just updated the file in git.

**Comment by sun-flat-yamada** [comment]
Thank you for your confirmation, @maartenba .
However, I don't correct this case yet.

> ```
>          <shortcut id="Navigate To Related Symbol">Ctrl+Alt+F7</shortcut> (Rider Visual Studio keymap)
> ```

You described it as "Navigate to related files (Ctrl+Alt+F7)", but it is assigned to "Show usages" and will not work in this workshop.

[Show usages (Ctrl+Alt+F7) in Navigate from symbols](https://www.jetbrains.com/help/rider/Reference_Keymap_Rider.html?keymap=primary_visual_studio#navigate_from_symbols)

![jetbrains-rider-whats-the-right-way-to-do-navigate-to-related-files](https://user-images.githubusercontent.com/51235130/118643087-1457f780-b817-11eb-8e37-cd558a1950a3.png)

I have attached a recording of the error I get when I run "Show usages (Ctrl+Alt+F7)" in this workshop.

![jetbrains-rider-whats-the-right-way-to-do-navigate-to-related-files-02](https://user-images.githubusercontent.com/51235130/118643136-2174e680-b817-11eb-8a16-b6b4f44cea0f.png)

Do you get a different result in your environment?

**Comment by maartenba** [comment]
I use the VS keymap in Rider, can you check this action name/shortcut in your keymap?

![image](https://user-images.githubusercontent.com/485230/118650708-08ebd880-b7e5-11eb-8ad7-a3922878cd7e.png)

**Comment by sun-flat-yamada** [comment]
I was able to solve it.
There were two problems here.

1. The key assignment on the web help page is wrong: `Ctrl+Alt+F7` is assigned to Show usages in Rider Visual Studio style (as explained above).
2. In my environment, I was using Rider IntelliJ style. As you pointed out. Thanks.

As we discussed [here](https://github.com/JetBrains/resharper-rider-samples/issues/26#issuecomment-831101930
) before, I think it would be better to unify to post IntelliJ style in Rider workshop.

So I'll put out a revised commit in PR after.

Thanks, @maartenba .
